src/main/webapp/script.js

```diff
@@ -387,12 +388,19 @@ function getWeeklyView(aggregated) {
   thirtyDaysFromNow.setDate(thirtyDaysFromNow.getDate() - 30);
 
   var firstDayOfWeek = new Date(thirtyDaysFromNow);
-  firstDayOfWeek.setDate(firstDayOfWeek.getDate() - (today.getDay() - 1));
+  console.log(firstDayOfWeek);
+  console.log(firstDayOfWeek.getDay());
```

Comment:
We should also erase these printings :D

Reply:
Also good catch! I need to stop leaving print statements in 😅

Reply:
Fixed in 4b1a871.